tests/test-cases/Group23-VIC-Machine-Service/23-02-VCH-List.robot

```diff
@@ -111,3 +111,17 @@ Get VCH List Within Invalid Datacenter and Compute Resource
 
     Verify Return Code
     Verify Status Not Found
+
+    [Teardown]   Run Secret VIC Machine Delete Command   %{VCH-NAME}
+
+Empty VCH list returned
+    Get VCH List
+
+    Verify Return Code
+    Verify Status Ok
+
+    Get VCH ID    %{VCH-NAME}
```

Comment:
After the teardown keyword of the previous test case, the VCH is destroyed already, I'm not sure if getting the ID would work here in this case?